server/spec/hypervisor_check_in_spec.rb

```diff
@@ -216,8 +213,7 @@ def should_not_add_new_consumer_when_create_missing_is_false(result)
     #because mysql
     sleep 2
     job_detail = async_update_hypervisor(@owner, @consumer, @expected_host_name, @expected_host_hyp_id,  ['g1', 'g2'])
-    job_detail['result'].should == 'Created: 0, Updated: 1, Unchanged: 0, Failed: 0'
-    result_data = job_detail['resultData']
+    result_data = job_detail['result']
```

Comment:
I think virt-who is only looking for the `resultData` and `state`, and not for the `result` string summary, so it is probably OK not having this here, but I could be wrong. If I'm wrong, then virt-who will need a task to accordingly handle this change. @wottop Am I correct in this?^